config/deployments/integration-tests/crds/vmoperator.vmware.com_clustervirtualmachineimages.yaml

```diff
@@ -220,8 +220,14 @@ spec:
                   type: object
                 type: array
```

Comment:
Are these files still used? (Wondering because there's integration test in the name)

Fine to follow up in a separate PR if we still use it for e2e

Reply:
I think we should be able to drop them. E2E tests install these CRDs via the vm-operator's manifest.

Reply:
Just on my phone. But I think at least controller unit tests are using them

Reply:
I'm definitely fine with moving on with this PR. Maybe a follow up would be nice.

Although integration test might be appropriate as a folder name

Reply:
Got it, they are referenced at https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/b99561a50c54144e345f9d090015d8032a1f5744/controllers/vmware/test/controllers_suite_test.go#L81